lib/NS1_request.js

```diff
@@ -171,6 +187,9 @@ function create_promise() {
  * @private
  */
 function handle_error(err, response) {
+  if (errorCb) {
+    errorCb(err, response);
```

Comment:
We don't have to change any of the error handling behavior, we can just bolt the errorCb check right in where we were already handling errors.